src/test/ui/class-missing-self.stderr

```diff
@@ -2,7 +2,7 @@ error[E0425]: cannot find value `meows` in this scope
   --> $DIR/class-missing-self.rs:9:7
    |
 LL |       meows += 1;
-   |       ^^^^^ help: try: `self.meows`
+   |       ^^^^^ help: you might have meant to use the available field: `self.meows`
```

Comment:
(As the author of that "try", I still prefer less wordy labels.)

Reply:
The longer text is there for people that come from languages like Java, where implicit `this`. Having the longer text let us differentiate between fields and bindings when communicating with users.